### PR TITLE
Issue Five

### LINKED ISSUE #5: Familiarize with Development Environment and Tech Stack
Set up the development environment and gain a basic understanding of the project's tech stack. This includes:

- [x] Install and learn basic Git and GitHub operations
- [x] Set up VS Code (or preferred IDE) with relevant extensions
- [x] Review and understand the project's tech stack
- [x] Complete a simple exercise using the project's primary framework:
Set up the body layout for the home page, which includes button integration for bone anatomy sections (Bony Pelvis, Lower Limb, Skull, Upper Limb, Thorax, and Vertebrae) are integrated smoothly.

**Why it's useful:** Ensures new contributors have the necessary tools and foundational knowledge to start contributing effectively.
**Effort:** Can be completed in about 2 hours, depending on prior experience. Additional time may be needed for deeper exploration.

### PR DESCRIPTION
**Pull request by Balijah**
Fixes #5. Familiarized myself with the tech stack and created a simple dashboard with clickable buttons but they don't function just yet. Looking to expand on this before class if time allows.

### CODE REVIEW

**Review by tluke900 — COMMENTED** [review]
The comments on the styles file are every helpful, especially when clarifying what color your hex maps too.

File: frontend/styles.css
Comment (JoshBudzynski) [inline frontend/styles.css:1]: Try using the same coloring method. RGB vs Color code

**Comment by DeviPriya-Kolla** [comment]
Good start on the layout, but there's a key issue to address:

**1. Main concern:** You didn't use the skeleton code/layout provided by the tech lead.

**2. Why this matters:**
   - Could cause integration problems later
   - Might not follow project standards
   - May require significant rework

**3. What to do next:**
   - Review the original skeleton code
   - Redo your work based on that skeleton
   - Keep your current ideas, but apply them to the provided structure

**4. For future tasks:**
   - Always start with any provided base code
   - If you have concerns about the base code, discuss with the tech lead first

**5. Positive notes:**
   - Good job on creating a functional layout!!!
   - Your initiative is appreciated, but needs to align with project guidelines

Remember, following the project's established structure is crucial. Let's work on integrating your ideas within the existing framework.

**Review by DeviPriya-Kolla — COMMENTED** [review]
Good start on the layout, but there's a key issue to address:

**1. Main concern:** You didn't use the skeleton code/layout provided by the tech lead.

**2. Why this matters:**
Could cause integration problems later
Might not follow project standards
May require significant rework

**3. What to do next:**
Review the original skeleton code
Redo your work based on that skeleton
Keep your current ideas, but apply them to the provided structure

**4. For future tasks:**
Always start with any provided base code
If you have concerns about the base code, discuss with me first

**5. Positive notes:**
Good job on creating a functional layout!!!
Your initiative is appreciated, but needs to align with project guidelines

Remember, following the project's established structure is crucial. Let's work on integrating your ideas within the existing framework.

**Comment by Balijah** [comment]
Fixes #5. Integrated my code with the skeleton layout provided by the tech lead.

**Pull request closed without merging**